mel_jamei/week_01/thursday/homework03.js

```diff
@@ -0,0 +1,212 @@
+/*Geometry Function Lab
+
+Part 1, Rectangle
+
+Given the following a rectangle object like the one below, write the following functions:
+
+isSquare - Returns whether the rectangle is a square or not*/
+var rectangle = {
+  length: 3,
+  width: 4
+};
+
+var isSquare = function ( rect ) {
+
+    if (rect.length === rect.width ) {
+
+      console.log('this is a square');
+
+      return true;
+
+  } else {
+
+      console.log('this is not a square');
+  }
+};
+//area - Returns the area of the rectangle
+
+var area = function ( rectan ) {
+
+    var total = rectan.length * rectan.width;
+
+    return total;
+}
+
+/*perimeter - Returns the perimeter of the rectangle
+
+};*/
+
+var perimeter = function ( input ) {
+
+    //p = 2 * (length + width)
+
+    var total =  2 * (input.length + input.width);
+
+    return total;
+
+}
+
+isSquare(rectangle);
+area(rectangle);
+perimeter(input);
+
+
+//Part 2, Triangle
+
+//Given the following a triangle object like the one below, write the following functions:
+
+
+//isEquilateral - Returns whether the triangle is equilateral or not
+
+var triangle = {
+  sideA: 3,
+  sideB: 4,
+  sideC: 4
+};
+
+
+
+var isEquilateral = function (triangle) {
+
+  var a = triangle.sideA;
+  var b = triangle.sideB;
+  var c = triangle.sideC;
+
+  if (a === b && b === c && a === c) {
+
+      console.log('This is an equilateral triangle');
+
+  } else {``
+
+      console.log('This is not an equilateral triangle');
+  }
+}
+
+//isIsosceles - Returns whether the triangle is isosceles or not
+
+var isIsosceles = function (triangle) {
+
+  var a = triangle.sideA;
+  var b = triangle.sideB;
+  var c = triangle.sideC;
+
+    if (!(a===b)c){
```

Comment:
This line is not going to do what you want. What is the `c` doing at the end there?